py3status/modules/deadbeef.py

```diff
@@ -3,95 +3,101 @@
 Display song currently playing in deadbeef.
 
 Configuration parameters:
-    cache_timeout: how often we refresh usage in seconds (default 1)
-    delimiter: delimiter character for parsing (default '¥')
-    format: see placeholders below (default '{artist} - {title}')
+    cache_timeout: refresh interval for this module (default 1)
+    delimiter: delimiter between metadata fields (default '¥å¥å¥å')
+    format: display format for this module (default '{artist} - {title}')
 
 Format placeholders:
-    {artist} artist
-    {title} title
-    {elapsed} elapsed time
-    {length} total length
-    {year} year
-    {tracknum} track number
+    {artist}        artist
+    {length}        length
+    {playback_time} playback time
+    {title}         title
+    {tracknumber}   track number
+    {year}          year
 
 Color options:
-    color_bad: An error occurred
+    color_degraded: Paused
+    color_good: Playing
+    color_bad: Not Playing
 
 Requires:
-        deadbeef:
+    deadbeef: a GTK+ audio player for GNU/Linux
 
 @author mrt-prodz
 """
-from subprocess import check_output, CalledProcessError
+
+import subprocess
```

Comment:
First try. `DB 0.7.0`. This error seems related to only launching deadbeef at this time.
```
lasers~/src/py3status/py3status/modules git:(deadbeef✗) python2 deadbeef.py
{'color': None, 'full_text': '', 'cached_until': 1490967099.0}
Traceback (most recent call last):
  File "deadbeef.py", line 114, in <module>
    module_test(Py3status)
  File "/home/chris/src/py3status/py3status/module_test.py", line 52, in module_test
    print(getattr(module, method)(*method_args))
  File "deadbeef.py", line 79, in deadbeef
    out = self.py3.command_output(['deadbeef', '--nowplaying-tf', fmt])
  File "/home/chris/src/py3status/py3status/py3.py", line 810, in command_output
    raise Exception(msg.format(cmd=command[0], error=error))
Exception: Command 'deadbeef' had error starting deadbeef 0.7.0

lasers~/src/py3status/py3status/modules git:(deadbeef✗) echo $?
1
```